pkg/storage/chunk/azure/blob_storage_client_test.go

```diff
@@ -79,45 +75,19 @@ func Test_Hedging(t *testing.T) {
 					}),
 				}
 			}
-
-			c, err := NewBlobStorage(
-				&BlobStorageConfig{
-					AccountName: "account",
-					Environment: azureGlobal,
-					MaxRetries:  0,
-				},
-				metrics,
+			c, err := NewBlobStorage(&BlobStorageConfig{
+				ContainerName: "foo",
+				Environment:   azureGlobal,
+				MaxRetries:    1,
+			}, metrics,
 				hedging.Config{
 					At:           tc.hedgeAt,
 					UpTo:         tc.upTo,
 					MaxPerSecond: 1000,
 				})
 			require.NoError(t, err)
-
 			tc.do(c)
-			require.Eventually(t, func() bool { return tc.expectedCalls == count.Load() }, time.Second, time.Millisecond, "expected calls %d, got %d", tc.expectedCalls, count.Load())
+			require.Equal(t, tc.expectedCalls, count.Load())
 		})
 	}
 }
-
-func Test_IsObjectNotFoundErr(t *testing.T) {
-	c, err := NewBlobStorage(
```

Comment:
We don't need this test anymore ?

Reply:
We don't. I tried keeping the test to guard against 404s when we upgrade again but in this version of the SDK, it is just a compilation error.